crates/oxc_codegen/tests/integration/snapshots/pure_comments.snap

```diff
@@ -241,7 +241,7 @@ const defineSSRCustomElement = /* @__NO_SIDE_EFFECTS__ */ (
 };
         
 ----------
-const defineSSRCustomElement = /* #__NO_SIDE_EFFECTS__ */ (options, extraOptions) => {
+const defineSSRCustomElement = /* #__NO_SIDE_EFFECTS__ */ /* @__NO_SIDE_EFFECTS__ */ (options, extraOptions) => {
```

Comment:
I am not sure able to keep all annotation comments. Can you take a look this? @IWANABETHATGUY

Reply:
It's just for prettier codegen, I follow the esbuild before, https://esbuild.github.io/try/#YgAwLjIwLjIALS1idW5kbGUgLS1taW5pZnk9ZmFsc2UgLS1mb3JtYXQ9ZXNtAGUAZW50cnkuanMAZXhwb3J0IGNvbnN0IGEgPSAvKiAjX19OT19TSURFX0VGRkVDVFNfXyAqLyAvKiAjX19OT19TSURFX0VGRkVDVFNfXyAqLyAoKSA9PiB7Cn0AAGZpbGUuanMAdmFyIHYgPSB0eXBlb2YgbG9jYWxWYXIKbW9kdWxlLmV4cG9ydHMgPSB2

Reply:
OK, Thanks!